README.md

```diff
@@ -21,34 +21,29 @@ requirements of the defined Myna workflow steps
 
 ## Installation
 
-Myna is a Python package which requires Python 3. You can install it locally using `pip install .`,
-since the package is not on PyPI. To install `myna` follow the instructions below.
+Myna is a Python package which requires Python 3. You must install it locally using
+`pip install -e .`, since the package is not on PyPI. To install `myna` follow the
```

Comment:
... This is a temporary requirement and/or will change in future releases ...

Reply:
Updated the wording